editoast/src/schema/rolling_stock.rs

```diff
@@ -110,6 +110,100 @@ impl RollingStock {
     }
 }
 
+#[derive(Debug, Queryable, Serialize)]
```

Comment:
To avoid having de declare all fields to select.

```suggestion
#[derive(Debug, Queryable, Serialize, Selectable)]
```